doc/integrator/authentication.rst

```diff
@@ -23,6 +23,11 @@ In the file ``env.sample``, you can configure the policy with the following vari
    AUTHTKT_SECURE  # Default to true.
    AUTHTKT_SAMESITE  # Default to Lax.
 
+.. note::
+
+   With the default configuration, for security reason the authentication will only works it the project is
```

Comment:
With the default configuration, for security reasons, the authentication will only work if the project is

Reply:
Thanks :-)

Reply:
Thank you for the enhancement